src/Components/LocationModal.tsx

```diff
@@ -1,34 +1,170 @@
+  faUserMd,
+  faVial,
+  faTasks,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
 import { labelMap } from '../App';
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    modal: {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    card: {
+      maxWidth: '90%',
+      width: '600px',
+      maxHeight: '90%',
+      overflowY: 'auto',
+    },
+    bottomNavigation: {
+      '&.Mui-selected': {
+        color: indigo[800],
+      },
+    },
+    media: {
+      height: 0,
+      paddingTop: '56.25%', // 16:9
+    },
+    detailText: {
+      marginLeft: 'auto',
+    },
+    expand: {
+      transform: 'rotate(0deg)',
+      marginLeft: 'auto',
+      transition: theme.transitions.create('transform', {
+        duration: theme.transitions.duration.shortest,
+      }),
+    },
+    expandOpen: {
+      transform: 'rotate(180deg)',
+    },
+    avatar: {
+      backgroundColor: red[500],
+    },
+    callToAction: {
+      borderRadius: '20px',
+      width: '100%',
+      height: '60px',
+      fontSize: '20px',
+      color: theme.palette.getContrastText(indigo[800]),
+      backgroundColor: indigo[800],
+      '&:hover': {
+        backgroundColor: indigo[900],
+        // Reset on touch devices, it doesn't add specificity
+        '@media (hover: none)': {
+          backgroundColor: indigo[800],
+        },
+      },
+    },
+    grid: {
+      flexGrow: 1,
+    },
+    cardMargin: {
+      marginBottom: '16px',
+    },
+    detailsButton: {
+      paddingLeft: '8px',
+    },
+    cardActions: {
+      cursor: 'pointer',
+    },
+    cardHeader: {
+      paddingBottom: '0px',
+    },
+    typeChip: {
+      backgroundColor: orange[900],
+      color: theme.palette.getContrastText(indigo[800]),
+      height: '20px',
+      marginTop: '5px',
+    },
+  })
+);
+
 interface ModalProps {
   location: any;
   onClose: Function;
 }
 
+interface LocationIconProps {
+  locationType: any;
+}
+
 const LocationModal = ({ location, onClose }: ModalProps) => {
+  // const [value, setValue] = React.useState(0);
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  const classes = useStyles();
+  function renderLocationIcon(param: any): IconProp {
+    switch (param) {
+      case 'Urgent Care':
+        return faStethoscope;
+      case 'Medical Center':
+        return faHospital;
+      case 'Health Center':
+        return faFirstAid;
+      case 'Clinic':
+        return faClinicMedical;
+      case 'Primary Care':
+        return faUserMd;
+      case 'Temporary':
+        return faCampground;
+      case 'Immediate Care':
+        return faMedkit;
+      case 'Public Health Department':
+        return faShieldAlt;
+      default:
+        return faHospital;
+    }
+  }
```

Comment:
This method could go in a utils file or something similar, doesn't need to be in the component